packages/flame_riverpod/pubspec.yaml

```
@@ -0,0 +1,19 @@
+name: flame_riverpod
+description: Model classes and an example project for using Riverpod in conjunction with Flame.
+version: 4.0.0+2
+homepage: https://markvideon.dev
```

```suggestion
homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_riverpod
```
You can put your own page in the readme if you want though :)